static/js/stream_list.js

```diff
@@ -517,6 +536,18 @@ export function set_event_handlers() {
         e.stopPropagation();
     });
 
+    $("#stream_filters").on("click", "li .toggle_stream_mute", (e) => {
```

Comment:
```javascript
$("#stream_filters").on("click", "li .toggle_stream_mute", (e) => {
```
I think `.toggle_stream_mute` should be `.toggle_stream_muted`

Reply:
@m-e-l-u-h-a-n Since we are not adding `Mute icon` near muted streams is this part necessary?

Reply:
Actually I could not find any element with class `.toggle_stream_mute` in dev tools inspector. Can you please mention about which mute icon you are setting event handler for?